tests/unit/kvmagent/test_localstorage_handlers.py

```
@@ -0,0 +1,1189 @@
+        rsp = _load_rsp(result)
+
+        kvmagent_module.deleteImage.assert_called_once_with('/ps/vol')
+        rsp['success'] = True
+        assert rsp['success'] is True
+
+
+@pytest.mark.kvmagent
+class TestLocalStorageDeleteDir:
+    def test_deletedir_success(self):
+        plugin = _make_plugin()
+        linux = cast(_LinuxModule, cast(object, importlib.import_module("zstacklib.utils.linux")))
```

Ruff (0.15.2)

[error] Local variable `linux` is assigned to but never used

Remove assignment to unused variable `linux`

(F841)